app/src/components/Navbar/i18n.json

```diff
@@ -57,9 +57,12 @@
         "userMenuOperationalToolbox":"Operational Toolbox",
         "userMenuCatalogueSurgeServices":"Catalogue of Surge services",
         "userMenuLearnLabel":"Learn",
+        "userMenuOperationalLearnings":"Operational Learnings",
         "userMenuTools":"Tools",
         "userMenuResources":"Resources",
         "userMenuGoBlog":"GO Blog",
+        "userMenuOperationalLearningsItem":"Operational Learnings",
+        "userMenuOperationalLearningsItemDescription":"Operational learnings in emergencies are the lessons learned from managing and dealing with crises, refining protocols for resource allocation, decision-making, communication strategies, and others.",
```

Comment:
Let's use a proper name to define variables. 

Here, `operationalLearningsDescription` makes sense.